backend/tests/unit/services/test_alert_service_pause_resume.py

```
@@ -0,0 +1,308 @@
+                timestamp_us=1234567890,
+                final_analysis="Analysis completed after resume"
+            )
+        )
+        
+        # Mock MCP client factory
+        with patch.object(alert_service, 'mcp_client_factory') as mock_mcp_factory:
+            mock_mcp_factory.create_client = AsyncMock()
+            
+            # Mock event publishing
+            with patch('tarsy.services.events.event_helpers.publish_session_resumed', new=AsyncMock()) as mock_resume_event, \
+                 patch('tarsy.services.events.event_helpers.publish_session_completed', new=AsyncMock()) as mock_complete_event, \
```

Ruff (0.14.4)

Local variable `mock_complete_event` is assigned to but never used

Remove assignment to unused variable `mock_complete_event`

(F841)